lib/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,38 @@
+import { useId } from 'react';
+
+export interface InputFieldProps {
+  value: string;
+  onChange: (newValue: string) => void;
+  placeholder?: string;
+  label?: string;
+  help?: string;
+}
+
+export const InputField = ({ value, onChange, placeholder, label, help }: InputFieldProps) => {
+  const inputId = useId();
+  const helpId = useId();
+  return (
+    <>
+      {label && (
+        <label htmlFor={inputId} className="mb-4 inline-block align-top text-form-label text-primary-gray">
+          {label}
+        </label>
+      )}
+      <input
+        id={label ? inputId : undefined}
+        type="text"
+        value={value}
+        onChange={(event) => onChange(event.target.value)}
+        placeholder={`(${placeholder})`}
```

Comment:
This is done like in the designs at the moment, but the placeholder is quite dark and it seems not so clear that it is just a placeholder value except the parens.

I hope these placeholders are not used/used in minimum as the help will vanish when user starts typing. 
Would be user friendlier to have the hint always visible, e.g between the label and the input (as the current help text is under the field and it might change if there is error and not usable in this case).

Reply:
It has done as in the design. Agree and hopefully that will change at some point. However, this discussion should be with the UX designer.